manifests/init.pp

```diff
@@ -38,13 +38,28 @@
 #       relayhosts    => ['[mail01.example.com]','[mail02.example.com]:2525'],
 #       myorigin      => 'server.example.com'
 #     }
+# * Installation of postfix with smtp relay authentication
+#     class {'postfix':
+#       root_email               => 'email@example.com',
+#       relayhosts               => ['[mail01.example.com]','[mail02.example.com]:2525'],
+#       myorigin                 => 'server.example.com'
+#       smtp_relay_auth_enabled  => true
+#       smtp_relay_auth_email    => user@example.com
+#       smtp_relay_auth_password => secret_password
+#
+#     }
 #
```

Comment:
can the above be cleaned up?

Reply:
this is a documentation, part of https://github.com/skyscrapers/puppet-postfix/blob/master/manifests/init.pp#L33